apollo-gradle-plugin/src/main/kotlin/com/apollographql/apollo/gradle/internal/DefaultService.kt

```diff
@@ -123,16 +128,7 @@ abstract class DefaultService @Inject constructor(val objects: ObjectFactory, ov
         """
 Multiple schemas found:
 ${candidates.joinToString(separator = "\n")}
```

Comment:
This error is now missing a suggestion. When we fail because of multiple schemas, what should they do?

Reply:
The answer to this question depends whether you're using `service` or `createAllAndroidVariantServices`. It was actually wrong to suggest using multiple services before if the multiple schemas were the result of schemas being present in multiple flavor/buildType source sets. So I'd prefer a less precise error message over a wrong one.

Reply:
Can we say that multiple schema files are not supported?

Reply:
I added this: `Multiple schemas are not supported. You can either define multiple services or specify the schema you want to use explicitely with `schemaFile`. I think this is technically not wrong :)